### ISSUE TITLE
code bug

### ISSUE THREAD

**Issue by John122-art** [jchengai/forecast-mae#21]
python3 train.py data_root=/path/to/data_root model=model_forecast gpus=4 batch_size=32 monitor=val_minFDE 'pretrained_weights="/path/to/pretrain_ckpt"'

When I fine-tune the model using the command above, I encountered a bug. After my verification, I found that "monitor=val_minFDE" caused the error. However, when I changed it to "monitor=val_minFDE6", it worked fine. Perhaps it's due to my lack of knowledge in this area, which led me to think this was a bug. Could you please help me understand why this happened? I would greatly appreciate it!
![image](https://github.com/user-attachments/assets/9d84275d-5c90-49bc-a733-388f3aa7650e)

**Comment by zzt007** [comment]
Hello, I also encountered the same problem. I think the run command in the readme is just an example; in fact, It's actually because there's no such metric; you just need to replace it with one of the optional metrics,  w.r.t val_minFDE6 \ val_minFDE1 ...
